index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Grow a Digitree</title>
+	<link rel="stylesheet" href="">
+</head>
+<body>
+
+	<form action="" method="get" accept-charset="utf-8">
+		<fieldset>
+			<legend><h1>Grow a Digitree</h1></legend>
```

Comment:
header tags are not allowed in legend tags, only tags that are considered "Phrasing Content" like `<span>`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content